stdlib/socketserver.pyi

```diff
@@ -1,8 +1,8 @@
 import sys
 import types
-from _typeshed import Self
+from _typeshed import ReadableBuffer, Self
 from collections.abc import Callable
-from socket import socket as _socket
+from socket import _RetAddress, socket as _socket
```

Comment:
Nit: can we import `_RetAddress` from `_socket` rather than `socket`, so I only have to open one file to find out what the alias is, instead of opening up `socket.pyi`, finding it's actually re-exported from `_socket.pyi`, then opening `_socket.pyi`? 🙃

Reply:
Done